GUIs/ScriptRunnerPyQt5_GUI/scripFunc/scripWINONLY.py

```diff
@@ -327,3 +323,436 @@ def browserCONF(self):
+                            completedPOP()
+                        elif self.adminyn == 'n':
+                            print('This user will not be an admin')
+                            command = """
+$nusnm = """ + '"{}"'.format(username) + """
+$nuspss = ConvertTo-SecureString """ + '"{}"'.format(passwd) + """ -AsPlainText -Force
+New-LocalUser -Name $nusnm -Password $nuspss"""
+                            print(command)
+                            sub.Popen(["powershell", "& {" + command + "}"])
+                            print('User ' + username + " has been successfully added!")
+                            completedPOP()
+
+                def cancelbutton():
+                    self.close()
+
+                self.Confirm_button.clicked.connect(lambda: threader(CONFIRM()))
+                self.Cancel_button.clicked.connect(cancelbutton)
+
+                def threader(com):
+                    try:
+                        threader = Thread(target=com)
+                        threader.start()
+                    except Exception as e:
+                        print(e)
+                        print('Could not start thread')
+
+        def calladdusrtosys():
+            widget = addusrtosys()
+            widget.exec_()
+
+        calladdusrtosys()
+
+    def remusr(self):
+        pass
+
+    def addgrutosys(self):
+        pass
+
+    def remgrufrosys(self):
+        pass
+
+    def addusrtogru(self):
+        pass
+
+    def remusrfrogru(self):
+        pass
+
+    def lslocausrs(self):
+        pass
+
+    def lslocagrus(self):
+        pass
+
+    def lsmemofgru(self):
+        pass
+
+    def lsgrusanusrin(self):
+        pass
+
+    # @staticmethod
+    def chngpasswdofall(self):
+
+        # TODO: This command as of July 7th, 2020 is broken. More on the issue can be found here: https://github.com/vipersniper0501/CP_Scripts2/issues/48
+
+        class changepasswordofall(QDialog, Ui_chngpass):
+            def __init__(self, parent=None):
+                super(changepasswordofall, self).__init__(parent)
+                self.setWindowIcon(QtGui.QIcon(':/Pictures/images/cup2.png'))
+                self.setupUi(self)
+                self.EXECUTE()
+
+            def EXECUTE(self):
+                def findnames():
+                    command = 'Get-LocalUser'
+                    EXEC = sub.Popen(["powershell", "& {" + command + "}"], stdout=sub.PIPE)
+                    stdout, _ = EXEC.communicate()
+                    output = stdout.decode("utf-8")
+                    output = output.split('\n')
+                    print(output)
+                    n = []
+                    for i in range(0, len(output)):
+                        if 'True' in output[i]:
+                            # print('found in ' + str(i))
+                            # print(output[i])
+                            n.append(output[i])
+                    a = []
+                    for i in range(0, len(n)):
```

Comment:
Codacy found an issue: [Consider using enumerate instead of iterating with range and len](https://app.codacy.com/manual/vipersniper0501/CP_Scripts2/pullRequest?prid=5823742)